### ISSUE TITLE
doctype= editorial

### ISSUE THREAD

**Issue by golnazads** [adsabs/export_service#159]
add editorial to all formats

**Comment by golnazads** [comment]
https://github.com/adsabs/export_service/blob/master/exportsrv/formatter/bibTexFormat.py#L135
https://github.com/adsabs/export_service/blob/master/exportsrv/formatter/fieldedFormat.py#L54

@aaccomazzi for fielded formats, should editorial be mapped to journal the same as erratum and obituary? 

        if (export_format == self.EXPORT_FORMAT_ENDNOTE):
            fields = {... 'erratum': 'Journal Article', 'obituary': 'Journal Article'}
        elif (export_format == self.EXPORT_FORMAT_PROCITE):
            fields = {... 'erratum': 'Journal', 'obituary': 'Journal'}
        elif (export_format == self.EXPORT_FORMAT_REFMAN):
            fields = {... 'erratum': 'JOUR', 'obituary': 'JOUR'}
        elif (export_format == self.EXPORT_FORMAT_REFWORKS):
            fields = {... 'erratum': 'Journal', 'obituary': 'Journal'}
        elif (export_format == self.EXPORT_FORMAT_MEDLARS):
            fields = {... 'erratum': 'Journal Article', 'obituary': 'Journal Article'}

**Comment by golnazads** [comment]
aaccomazzi  12:08 PM
a doctype of `editorial` should be `@article` in bibtex.  
```
        fields = {'article':'@ARTICLE', 'circular':'@ARTICLE', 'newsletter':'@ARTICLE',
                  'bookreview':'@ARTICLE', 'erratum':'@ARTICLE', 'obituary':'@ARTICLE',
                  'eprint':'@ARTICLE', 'catalog':'@ARTICLE',
                  'book':'@BOOK', 
                  'inbook':'@INBOOK',
                  'proceedings':'@PROCEEDINGS', 
                  'inproceedings':'@INPROCEEDINGS', 'abstract':'@INPROCEEDINGS',
                  'misc':'@MISC', 'software':'@MISC','proposal':'@MISC', 'pressrelease':'@MISC',
                  'talk':'@MISC',
                  'phdthesis':'@PHDTHESIS','mastersthesis':'@MASTERSTHESIS',
                  'techreport':'@TECHREPORT', 'intechreport':'@TECHREPORT'}
```
Also, we should have a default rule in terms of mapping doctypes to bibtex types, right now the output is broken.  I suggest defaulting to @misc
@aaccomazzi I do not agree with this, I feel that we need to have a control over doctype, Not producing output when service does not recognize a doctype makes it is easier to find out a new doctype has been introduced that service has to consider. I shall bring it up in weekly meeting today.

**Comment by golnazads** [comment]
implemented in v2.0.65.
